tasmota/language/en-GB.h

```diff
@@ -628,6 +628,7 @@
 #define D_SENSOR_SLAVE_TX    "Slave TX"
 #define D_SENSOR_SLAVE_RX    "Slave RX"
 #define D_SENSOR_SLAVE_RESET "Slave RST"
+#define D_SENSOR_GP2Y10      "GP2Y10"
```

Comment:
This key is missing for the rest of the Language files making travis test to fail.